chapters/ch20.md

```diff
@@ -8,15 +8,15 @@ Giả sử khi huấn luyện, tập phát triển và tập kiểm tra có cùn
 
 > Even though having more data can’t hurt, unfortunately it doesn’t always help as much as you might hope. It could be a waste of time to work on getting more data. So, how do you decide when to add data, and when not to bother?
 
-Mặc dù có thêm dữ liệu không thể giảm hiệu quả, thật không may điều này không luôn luôn mang lại hiệu quả nhiều như bạn mong đợi. Việc thu thập thêm dữ liệu có thể trở nên lãng phí thời gian. Vậy làm thể nào để quyết định khi nào nên thêm dữ liệu, khi nào không nên?
+Mặc dù có thêm dữ liệu không làm tổn hại đến hiệu quả, thật không may điều này không phải lúc nào cũng mang lại hiệu quả nhiều như bạn mong đợi. Việc thu thập thêm dữ liệu có thể trở nên lãng phí thời gian. Vậy làm thể nào để bạn quyết định khi nào nên thêm dữ liệu, khi nào không nên?
```

Comment:
Câu đầu tiên lặp hai từ "hiệu quả", anh vẫn chưa biết giải quyết sao.

Từ "you" trong tiếng Anh có thể có nhưng khi dịch mình có thể bỏ. Sao cho đơn giản, "thuần" Việt,  mà không làm thay đổi nghĩa.
```suggestion
Mặc dù có thêm dữ liệu không làm tổn hại đến hiệu quả, thật không may, điều này không phải lúc nào cũng mang lại hiệu quả nhiều như mong đợi. Việc thu thập thêm dữ liệu có thể trở nên lãng phí thời gian. Vậy làm thể nào để bạn quyết định khi nào nên thêm dữ liệu, khi nào không nên?
```